src/Conda.jl

```diff
@@ -35,42 +35,98 @@ using Compat
 import Compat.String
 using JSON
 
-"Prefix for installation of all the packages."
-const PREFIX = abspath(dirname(@__FILE__), "..", "deps", "usr")
+deps_file = joinpath(dirname(@__FILE__), "..", "deps","deps.jl")
+
+if isfile(deps_file)
+    include(deps_file)
+else
+    error("Conda is not properly configured.  Run Pkg.build(\"Conda\") before importing the Conda module.")
+end
+
+typealias Environment Union{AbstractString,Symbol}
+
+RootEnv = default_dir
```

Comment:
why not just use `default_dir` here?  Or call it `rootenv` in the `deps.jl` file?

(Note that CamelCase is used only for types in the typical Julia style.)